playbooks/files/grafana/grafana.ini.j2

```diff
@@ -0,0 +1,33 @@
+# {{ ansible_managed }}
+
+[server]
+http_port = {{ grafana__server__http_port }}
+domain = {{ grafana__server__domain }}
+root_url = {{ grafana__server__root_url }}
+
+[analytics]
+reporting_enabled = {{ grafana__analytics__reporting_enable }}
```

Comment:
```suggestion
reporting_enabled = {{ grafana__analytics__reporting_enabled }}
```

Reply:
Also in group_vars/grafana_servers/00-defaults.yaml